backend/src/openbeheer/zaken/views.py

```diff
@@ -0,0 +1,92 @@
+import uuid
+
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema
+from msgspec.json import decode
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from openbeheer.api.views import MsgspecAPIView
+from openbeheer.clients import zrc_client
+from openbeheer.types import (
+    OBField,
+    OBFieldType,
+    OBList,
+    OBPagination,
+    OBSelection,
+    Zaak,
+    ZGWError,
+    ZGWResponse,
+    as_ob_option,
+)
+
+
+@extend_schema(
+    summary=_("Zaken"),
+)
+class ZakenView(MsgspecAPIView):
```

Comment:
See comment on the client. -> Do we need a Zaak view?

Reply:
As discussed: we should focus on zaaktype not zaken